go.mod

```diff
@@ -215,14 +216,14 @@ require (
 	go.opentelemetry.io/otel/trace v1.16.0
 	go.uber.org/atomic v1.11.0
 	go.uber.org/goleak v1.2.1
-	golang.org/x/crypto v0.14.0
```

Comment:
Out of curiosity, are these updated with the updated entry for `hcp-sdk-go`?

Reply:
Yes